### PR TITLE
LPS-123218 1st batch of SAML changes (persistence changes)

### PR DESCRIPTION
**Pull request by stian-sigvartsen**
Hi @martamedio . I think we should actually send this first batch under 2 tickets because e91c44f41bda causes a major version bump, and we can get away with not merging it to 7.3 . It would just be a slightly degraded functionality. I'll start a thread on Slack.

### CODE REVIEW

File: modules/dxp/apps/saml/saml-persistence-api/src/main/java/com/liferay/saml/persistence/model/SamlPeerBindingSoap.java [hunk 1]
```diff
@@ -0,0 +1,222 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-persistence-api/src/main/java/com/liferay/saml/persistence/model/SamlPeerBindingSoap.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-persistence-api/src/main/java/com/liferay/saml/persistence/model/SamlPeerBindingWrapper.java [hunk 2]
```diff
@@ -0,0 +1,440 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-persistence-api/src/main/java/com/liferay/saml/persistence/model/SamlPeerBindingWrapper.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-persistence-api/src/main/java/com/liferay/saml/persistence/service/SamlPeerBindingLocalServiceWrapper.java [hunk 3]
```diff
@@ -0,0 +1,382 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-persistence-api/src/main/java/com/liferay/saml/persistence/service/SamlPeerBindingLocalServiceWrapper.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-persistence-service/src/main/java/com/liferay/saml/persistence/model/impl/SamlPeerBindingCacheModel.java [hunk 4]
```diff
@@ -0,0 +1,264 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-persistence-service/src/main/java/com/liferay/saml/persistence/model/impl/SamlPeerBindingCacheModel.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-persistence-service/src/main/java/com/liferay/saml/persistence/model/impl/SamlPeerBindingModelImpl.java [hunk 5]
```diff
@@ -0,0 +1,1057 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-persistence-service/src/main/java/com/liferay/saml/persistence/model/impl/SamlPeerBindingModelImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/saml/saml-persistence-service/src/main/java/com/liferay/saml/persistence/service/persistence/impl/SamlPeerBindingPersistenceImpl.java [hunk 6]
```diff
@@ -0,0 +1,1748 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-persistence-service/src/main/java/com/liferay/saml/persistence/service/persistence/impl/SamlPeerBindingPersistenceImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by stian-sigvartsen** [comment]
ci:test:saml

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
Test fails look unrelated 👍

**Comment by martamedio** [comment]
ci:reevaluate:1097401_4820

**Comment by martamedio** [comment]
ci:reevaluate:1097401_4820

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:test:sf

**Comment by martamedio** [comment]
Please, see: https://github.com/brianchandotcom/liferay-portal/pull/101965

**Pull request closed without merging**